Packs/Netskope/Integrations/NetskopeEventCollector/NetskopeEventCollector.py

```diff
@@ -0,0 +1,316 @@
+                                      headerTransform=string_to_table_header)
+
+    return CommandResults(outputs_prefix='Netskope.Event',
+                          outputs_key_field='_id',
+                          outputs=events,
+                          readable_output=readable_output,
+                          raw_response=events)
+
+
+def v2_get_events_command(client: Client, args: Dict[str, Any], last_run: dict) -> CommandResults:
+    limit = arg_to_number(args.get('limit', 50))
+
+    events = client.get_events_request_v2(last_run, limit)
+    for event in events:
+        event['timestamp'] = timestamp_to_datestring(event['timestamp'] * 1000)
+
+    readable_output = tableToMarkdown('Events List:', events,
+                                      removeNull=True,
+                                      headers=['_id', 'timestamp', 'type', 'access_method', 'app', 'traffic_type'],
+                                      headerTransform=string_to_table_header)
+
+    return CommandResults(outputs_prefix='Netskope.Event',
+                          outputs_key_field='_id',
+                          outputs=events,
+                          readable_output=readable_output,
+                          raw_response=events)
+
+
+''' MAIN FUNCTION '''
+
+
+def main() -> None:  # pragma: no cover
+    params = demisto.params()
+
+    url = params.get('url')
+    api_version = params.get('api_version')
+    token = demisto.params().get('credentials', {}).get('password')
+    base_url = urljoin(url, f'/api/{api_version}/')
+    verify_certificate = not demisto.params().get('insecure', False)
+    proxy = demisto.params().get('proxy', False)
+    first_fetch = params.get('first_fetch')
+    max_fetch = params.get('max_fetch')
+    vendor, product = params.get('vendor', 'netskope'), params.get('product', 'netskope')
+
+    demisto.debug(f'Command being called is {demisto.command()}')
+    try:
+        client = Client(base_url, token, api_version, verify_certificate, proxy)
+
+        last_run = demisto.getLastRun()
+        if 'alert' not in last_run and 'application' not in last_run and 'audit' not in last_run \
+                and 'network' not in last_run:
+            last_run = arg_to_seconds_timestamp(first_fetch)
+            last_run = {
+                'alert': last_run,
+                'application': last_run,
+                'audit': last_run,
+                'network': last_run
+            }
+
+        if demisto.command() == 'test-module':
+            # This is the call made when pressing the integration Test button.
+            result = test_module(client, api_version, last_run)
+            return_results(result)
+
+        elif demisto.command() == 'netskope-get-events':
+            if api_version == 'v1':
+                return_results(v1_get_events_command(client, demisto.args(), last_run))
+            else:
+                return_results(v2_get_events_command(client, demisto.args(), last_run))
```

Comment:
Let's add an argument should_push_events (see other collectors as an example) such that if set to true events will be sent to XSIAM as well.